bin/migrate_acl_authz.py

```diff
@@ -211,11 +242,7 @@ def acl_to_authz(self, record):
                     self.projects[acl_item], acl_item
                 )
                 projects_found += 1
-            else:
-                # nothing worked, raise exception
-                raise EnvironmentError(
-                    "program or project {} does not exist".format(acl_item)
```

Comment:
should we keep this at least as a warning log?

Reply:
the warning would end up needing to be more generic b/c now it's not just that program/proj don't exist, but that there's no mapping either. I think the error log in a few lines sorta covers that? https://github.com/uc-cdis/indexd/pull/338/files/24af7553837da98ddf5a53259b91e4f87d23a936#diff-02f08e0cd101b79f6a25f6d6689008de374d7b6399a2a1291ee52faafc73aacdR247

Lemme know if you disagree

Reply:
basically if nothing is found in any of the cases then there'll be an error log

Reply:
ah ok, yeah that other log works